package-lock.json

```diff
@@ -1,3883 +1,8 @@
 {
   "name": "chartjs-chart-financial",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
I don't know that we want to go back and forth between lockfile versions with every commit. Is it possible fo you to upgrade to a newer version of `npm`?

Reply:
Sorry didn't realise. Done.